languages/woocommerce-moip-pt_BR.po

```diff
@@ -2,846 +2,910 @@ msgid ""
 msgstr ""
```

Comment:
Do not update .po or .mo files.
This will be auto generated before submit to w.org.